server/controllers/meeting.js

```diff
@@ -0,0 +1,20 @@
+const Meeting = require("../models/Meeting");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+exports.createMeeting = asyncHandler(async (req, res, next) => {
+  const { username, duration } = req.body;
+  const loggedInUser = await User.findOne({ username }).then((doc) => {
+    const meeting = Meeting.create({
+      user_id: doc._id,
+      duration: duration,
+    });
+  });
```

Comment:
We could separate this here. `await` pauses the execution so no concern with one task finishing before the other. 

```js
const loggedInUser = await User.findOne({ username });
const meeting = Meeting.create({ . . . });
```